distro/openshift-template/mt/apicurio-registry-mt-template.yaml

```diff
@@ -780,23 +472,13 @@ parameters:
 
 - name: TENANT_MANAGER_URL
   value: http://tenant-manager:8585
-- name: TENANT_MANAGER_CA_BUNDLE
```

Comment:
Actually, the CA bundle must be preserved.

Reply:
restored